znai/src/main/java/com/twosigma/znai/extensions/markup/MarkdownsIncludePlugin.java

```diff
@@ -57,10 +62,9 @@ public PluginResult process(ComponentsRegistry componentsRegistry,
         MarkupParser parser = componentsRegistry.defaultParser();
 
         markdowns = markdowns(dir).collect(Collectors.toList());
-        Stream<DocElement> elements = markdowns.stream().flatMap(p -> {
-            MarkupParserResult parserResult = parser.parse(markupPath, FileUtils.fileTextContent(p));
-            return parserResult.getDocElement().getContent().stream();
-        });
+        parserResults
```

Comment:
nitpicking on a line alignment. Do you mind to break it like
```
.map(...)
.collect(...)
```

Reply:
updated.